manage/manage.go

```diff
@@ -47,4 +49,10 @@ func openDatabase() {
 
 	// Open the connection to the database
 	datastore.OpenSysDatabase(datastore.Environ.Config.Driver, datastore.Environ.Config.DataSource)
+
+	// Opening the keypair manager to create the signing database
+	err := datastore.OpenKeyStore(datastore.Environ.Config)
```

Comment:
this change is needed for the migration code to get access to the key store for signing